app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -102,6 +105,19 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         .observeOn(mainThread())
         .takeUntil(screenDestroys)
         .subscribe { uiChange -> uiChange(this) }
+
+    illustrationImageView.setImageResource(illustrationResourceId())
+  }
+
+  private fun illustrationResourceId(): Int {
```

Comment:
Since we have only one resource to show, this looks fine for now. If we start adding more checks to show different images based on the date, we'll need to make this more generic.